adapters/buenos-aires.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+import cheerio from 'cheerio';
+import async from 'async';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+
+export const name = 'buenos aires';
+export function fetchData (source, callback) {
+  // list of requests used to get data
+  const tasks = generateTasks(source);
+
+  async.parallel(tasks, (err, response) => {
+    if (err) {
+      return callback(new Error('Failure to load data urls.'));
+    }
+
+    // wrap in try catch to handle possible errors
+    try {
+      const data = formatData(response);
+      if (data === undefined) {
+        return callback(new Error('Failure to parse data.'));
+      }
+      callback(null, data);
+    } catch (e) {
+      return callback(new Error('Unknown adapter error.'));
+    }
+  });
+}
+
+// makes a requests list of requests for each station to each pollutant
+const generateTasks = (source) => {
+  const linkRequests = [];
+  ['CENTENARIO', 'CORDOBA', 'LA BOCA'].forEach((station) => {
```

Comment:
Can we make this an array of objects like [{name: 'Centenario', number: 1, coordinates: {lat: 23, lon: 23}}, ...] so that we can avoid so many switch statements later? 

Also, is there any way to get this info programatically instead of hard coding it?